hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -752,17 +749,23 @@ public Builder setLayoutVersion(Option<TimelineLayoutVersion> layoutVersion) {
       return this;
     }
 
-    public Builder setProperties(Properties properties) {
-      this.props = properties;
+    public Builder setMetaserverConfig(Properties props) {
+      this.metaserverConfig = new HoodieMetaserverConfig.Builder().fromProperties(props).build();
```

Comment:
you just need a new method to take in metaserver config, why removing existing API `setProperties()` ?

Reply:
is it because `props` only introduced for metaserver client?